ts/features/newWallet/components/WalletCardsCategoryContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import { IOIcons, ListItemHeader, VSpacer } from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { WalletCard, walletCardComponentMapper } from "../types";
+
+type WalletCategoryStackContainerProps = {
+  iconName: IOIcons;
+  label: string;
+  cards: ReadonlyArray<WalletCard>;
+};
+
+/**
+ * This component handles the rendering of cards of a specific category.
+ * The component also handles logic behind card stacking and animations
+ */
+const WalletCardsCategoryContainer = ({
+  label,
+  iconName,
+  cards
+}: WalletCategoryStackContainerProps) => {
+  const isStacked = cards.length > 1;
```

Comment:
Is this the boolean value to view the cards stacked or unstacked? If I manually change this value to false should I get the cards unstacked?